docs/rules/prefer-reflect.md

```diff
@@ -54,12 +54,12 @@ Examples of **correct** code for this rule when used without exceptions:
 ```js
 /*eslint prefer-reflect: "error"*/
 
-Reflect.apply(undefined, args);
-Reflect.apply(null, args);
+Reflect.apply(obj.foo, undefined, args);
```

Comment:
I'm happy with that. I think potentially `foo` and `obj.foo` need different names, e.g. `foo` can become `myFunction` and `obj.foo` `obj.myMethod` perhaps?